docs/modules/ROOT/pages/tutorial/package.adoc

```diff
@@ -0,0 +1,92 @@
+= Tutorial: Writing and using Commodore Packages
+
+This tutorial will guide you through the steps to create and use a Commodore Package.
+Commodore Packages allow you to distribute configuration for one or more components.
+
+We will write a simple Package deploying https://github.com/projectsyn/component-cert-manager[Component cert-manager] and bundling some custom defaults.
```

Comment:
I'd probably write "package" in the text, instead of always captializing it, but no hard preferences either way.